e2e-tests/cypress/tests/integration/channels/messaging/message_deleted_on_reply_spec.js

```diff
@@ -54,40 +55,11 @@ describe('Messaging', () => {
                     and('not.contain', message);
             });
 
-            // # Send message
```

Comment:
Why are no longer checking if the draft is in the textbox?

Reply:
there was a change that will delete the drafts from other user , if the root post is deleted